src/Modules/Manage/Dnn.PersonaBar.Users/Providers/DataProviders/SqlDataProvider/Uninstall.SqlDataProvider

```diff
@@ -16,6 +16,9 @@ IF EXISTS (SELECT * FROM dbo.sysobjects where id = object_id(N'{databaseOwner}{o
 	DROP PROCEDURE {databaseOwner}{objectQualifier}Personabar_GetUsers
 GO
 
+IF EXISTS (SELECT * FROM sys.procedures WHERE object_id = object_id(N'{databaseOwner}[{objectQualifier}Personabar_GetUsersBySearchTerm]'))
+	DROP PROCEDURE {databaseOwner}[{objectQualifier}GetUsersBySearchterm]
```

Comment:
should be 
DROP PROCEDURE {databaseOwner}[{objectQualifier}Personabar_GetUsersBySearchterm]